packages/core/src/components/GridKeyboardNavigationContext/__stories__/useGridKeyboardNavigationContext.stories.helpers.tsx

```diff
@@ -0,0 +1,114 @@
+import { range } from "lodash-es";
+import React, { forwardRef, useMemo, useCallback, useRef, RefObject } from "react";
+import cx from "classnames";
+import { action } from "@storybook/addon-actions";
+import { Button, Flex } from "../..";
+import useGridKeyboardNavigation from "../../../hooks/useGridKeyboardNavigation/useGridKeyboardNavigation";
+import "./useGridKeyboardNavigationContext.stories.scss";
+import { GridKeyboardNavigationContext, useGridKeyboardNavigationContext } from "../GridKeyboardNavigationContext";
+
+const ELEMENT_WIDTH_PX = 72;
+const PADDING_PX = 24;
+
+const ON_CLICK = action("Selected");
+
+interface DummyNavigableGridProps {
+  itemsCount: number;
+  numberOfItemsInLine: number;
+  itemPrefix?: string;
+  disabled?: boolean;
+  disabledIndexes?: number[];
+  withoutBorder?: boolean;
+}
+
+export const DummyNavigableGrid = forwardRef<HTMLDivElement, DummyNavigableGridProps>(
+  (
+    { itemsCount, numberOfItemsInLine, itemPrefix, disabled, disabledIndexes, withoutBorder },
+    ref: RefObject<HTMLDivElement>
+  ) => {
+    const width = useMemo(() => numberOfItemsInLine * ELEMENT_WIDTH_PX + 2 * PADDING_PX, [numberOfItemsInLine]);
+    const items = useMemo(() => range(itemsCount).map(num => `${itemPrefix} ${num}`), [itemPrefix, itemsCount]);
+    const getItemByIndex = useCallback((index: number) => items[index], [items]);
+    const { activeIndex, onSelectionAction } = useGridKeyboardNavigation({
+      ref,
+      itemsCount,
+      numberOfItemsInLine,
+      getItemByIndex,
+      onItemClicked: ON_CLICK,
+      disabledIndexes
+    });
+    const onClickByIndex = useCallback((index: number) => () => onSelectionAction(index), [onSelectionAction]);
+    return (
+      <div
+        className={cx("use-grid-keyboard-dummy-grid-wrapper", { "without-border": withoutBorder })}
+        style={{ width }}
+        data-disabled={disabled}
+        ref={ref}
+        tabIndex={-1}
+      >
+        {items.map((item, index) => (
+          <Button
+            key={item}
+            kind={Button.kinds.SECONDARY}
+            onClick={onClickByIndex(index)}
+            disabled={disabled || disabledIndexes?.includes(index)}
+            className={cx("use-grid-keyboard-dummy-grid-item", { "active-item": index === activeIndex })}
+          >
+            {item}
+          </Button>
+        ))}
+      </div>
+    );
+  }
+);
+
+DummyNavigableGrid.defaultProps = {
+  itemPrefix: "",
+  disabled: false,
+  disabledIndexes: [],
+  withoutBorder: false
+};
```

Comment:
@itaim18 Let's use defaults in the component's signature :) Although it should work perfectly fine, in this project we define defaults in the signature

Reply:
😅